js/script.js

```diff
@@ -22,21 +22,30 @@ $(function() {
             var $columnCardList = $('<ul>').addClass('column-card-list');
             var $columnDelete = $('<button>').addClass('btn-delete').text('x');
             var $columnAddCard = $('<button>').addClass('add-card').text('Add a card');
+            var $columnSetCardName = $('<form>').addClass('set-card-name').hide();
+
+            $columnSetCardName.append('<input type="text" class="card-name" id="js-card-name">');
```

Comment:
id should be unique, when you add more than one column you id will exist few times on your website